src/router.c

```diff
@@ -56,6 +57,7 @@ static void fill_routed_request_id(char *buf, size_t buf_size, const void *addre
 	} else {
 		snprintf(buf, buf_size, "%x_%p", uuid, address);
 	}
+	++uuid;
```

Comment:
OMG, I really forget that?

Still, this is C, not C++, so it should be uuid++;

Reply:
changed to uuid++